mangarack-component-core/src/providers/dynastyreader/series.ts

```diff
@@ -0,0 +1,172 @@
+import * as mio from '../../default';
+import {createChapter} from './chapter';
+import {enhance} from '../enhance';
+
+const httpService = mio.dependency.get<mio.IHttpService>('IHttpService');
+const htmlService = mio.dependency.get<mio.IHtmlService>('IHtmlService');
+const providerDomain = 'http://dynasty-scans.com';
+const remapGenreType: mio.IDictionary = {
```

Comment:
I'm pretty sure this remap is incorrect. The idea is to take the definitions of the provider (dynasty reader) and remap them into the `GenreType` enumerator (so mangarack understands it). If the name matches, great, no remap is needed, but when it doesn't, a remap is needed. Sci-fi on mangafox, for example, remaps to Science Fiction (as per the enumerator). Now, you're remapping Adult to NSFW here, which is a genre type that does not exist in mangarack. Please check the full list of genres of dynasty reader and compare them with the GenreType enumerator.